Goals/Goals/DetailController.swift

```diff
@@ -23,9 +23,6 @@ class DetailController: UIViewController {
 
   override func viewDidLoad() {
     super.viewDidLoad()
-  }
-
-  override func viewWillAppear(_ animated: Bool) {
```

Comment:
Yes, thanks, switching between macOS and iOS dev always leaves me confused with these life cycle issues 😉